utilities/validations.py

```diff
@@ -107,3 +109,22 @@ def validate_date_range(**kwargs):
     elif (('end_date' and 'start_date' in kwargs) and
             kwargs['end_date'] < kwargs['start_date']):
         raise ValueError('Earlier date should be lower than later date')
+
+
+def validate_room_labels(**kwargs):
+    """
+    Function to validate the room label string type
+    :params room_labels
+    """
+    room_labels = kwargs.get('room_labels')
+    new_label = re.search("[{}:]", str(room_labels))
+    if new_label:
```

Comment:
Variable naming here could be better. `new_label` is not the best name for a boolean variable.

Reply:
Noted